source/shadow/README.md

```diff
@@ -45,4 +45,130 @@ Device Client would also perform configuration validation check when detecting t
 ```
 *It is important to note that providing input and output files is optional.  If they are not configured or left blank, "{welcome","aws-iot}" will be published to this shadow and the `default-sample-shadow-document`file will be created under this path `~/.aws-iot-device-client/sample-shadow/` by default*
 
+### Policy Permissions
+In order to use the Shadow feature the device must first have permission to connect to IoT Core.
+The device must also be able to publish, subscribe, and receive messages on the Shadow topics. You can read more [here](https://docs.aws.amazon.com/iot/latest/developerguide/device-shadow-mqtt.html).
+The example policy below demonstrates the minimum permissions required. Simply replace the `<region>` and `<accountId`> with the correct values.
```

Comment:
update this line `the minimum permissions required` to `the minimum permissions required for Shadow feature.`